package.json

```diff
@@ -49,8 +49,9 @@
   "scripts": {
     "tsc": "tsc",
```

Comment:
I know this is not needed if we run `yarn tsc` but it's just in case we want to add some default flags in the near future? maybe that's not even needed ... maybe having the `tsconfig.js` file is enough? maybe we should remove it jajaja